src/qibocal/protocols/characterization/allxy/allxy.py

```diff
@@ -29,7 +29,7 @@ class AllXYResults(Results):
     """AllXY outputs."""
 
 
-AllXYType = np.dtype([("prob", np.float64), ("gate", np.int64)])
+AllXYType = np.dtype([("prob", np.float64), ("gate", "<U18")])
```

Comment:
```suggestion
AllXYType = np.dtype([("prob", np.float64), ("gate", "U5")])
```
5 characters should be enough since the longest str will be something like "Xp-Xp".